python-modules/cis_hris_api/cis_hris_api/idp.py

```diff
@@ -0,0 +1,132 @@
+import json
+import logging
+
+from functools import wraps
+from flask import request
+from flask import _request_ctx_stack
+from six.moves.urllib.request import urlopen
+from jose import jwt
+
+from cis_hris_api.common import get_config
+from cis_hris_api.exceptions import AuthError
+
+logger = logging.getLogger(__name__)
+
+CONFIG = get_config()
+
+AUTH0_DOMAIN = CONFIG('auth0_domain', namespace='hris_api', default='auth-dev.mozilla.auth0.com')
+API_IDENTIFIER = CONFIG('api_identifier', namespace='hris_api', default='https://hris-api.sso.allizom.org')
```

Comment:
hris.api.sso.allizom.org would be closer to the api pattern, though, not biggy

Reply:
We can configure when we deploy.